tests/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.6)
+include(${CMAKE_SOURCE_DIR}/runtime.cmake)
+
+project (sakura-unittest)
+
+option(BUILD_SHARED_LIBS "Build shared libraries (DLLs)." OFF)
```

Comment:
https://github.com/sakura-editor/sakura/pull/311#discussion_r205955128
に同じ

Reply:
ビルド対象をMTで統一できるなら大元のCMakeList.txtでMTを強制してしまえばよいと思います。

実験で書いて動いたっぽいものから抜粋。

```C++
# CMake compile with /MT instead of /MD
# https://stackoverflow.com/questions/14172856/cmake-compile-with-mt-instead-of-md
set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /MT")
set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /MTd")
```

最初にこれを入れておけば、上書きしない限り MT になるようでした。